docs/UserGuide.md

```diff
@@ -3,7 +3,19 @@ layout: page
 title: User Guide
 ---
 
-Wishful Shrinking is a **desktop app for managing your diet, keeping track of your on-hand ingredients, recipes, as well as the food you’ve eaten (along with their calories)**. It is optimized for fast and efficient typist as it uses a **Command Line Interface (CLI)** with the added beauty of a Graphical User Interface (GUI).
+Welcome to the Wishful Shrinking User Guide! This user guide provides in-depth documentation on the installation process, step-by-step instructions for each feature and troubleshooting recommendations. Wishful Shrinking is available for the Linux, Unix, Windows and Mac OS operating systems.
+
+
+## Introducing Wishful Shrinking
+Wishful Shrinking your desktop diet manager. It is an app that helps you manage your on-hand ingredients, organise personal recipes and track your diet. Wishful Shrinking facilitates a healthier diet in three main ways: 
```

Comment:
is it wishful shrinking is your...

Reply:
Wishful Shrinkng- your desktop diet manager. Is this clearer?

Reply:
okok can!